### ISSUE TITLE
The test/info menu only tests for video

### ISSUE THREAD

**Issue by mathieulh** [chriz2600/DreamcastHDMI#12]
The test/info menu only tests for video, it does not test for audio issues, so if you do not have proper connections to the AICA DAC it will still show all hearts as if there was no issue, this happened to me (no audio over HDMI, analog worked fine) and I had to resolder all the joints to the AICA chip to fix it.

**Comment by chriz2600** [comment]
Good point, unfortunately due to a lack of input pins on the FPGA, audio signals are connected directly to the HDMI transmitter, but I will see, if there is a way to get any meaningful debug information from the HDMI transmitter.
